src/legacy/core_plugins/timelion/public/panels/timechart/schema.ts

```diff
@@ -17,25 +17,32 @@
  * under the License.
  */
 
-require('./flot');
+import './flot';
 import _ from 'lodash';
 import $ from 'jquery';
 import moment from 'moment-timezone';
+import { timefilter } from 'ui/timefilter';
+// @ts-ignore
 import observeResize from '../../lib/observe_resize';
+// @ts-ignore
 import { calculateInterval, DEFAULT_TIME_FORMAT } from '../../../common/lib';
-import { timefilter } from 'ui/timefilter';
+import { LegacyDependenciesPluginStart } from '../../shim';
```

Comment:
Not sure that we should have link to 'shim' from here. 
I think here should be: import { TimelionVisualizationDependencies } from '../plugin';

Please see example here: `src\legacy\core_plugins\vis_type_vega\public\plugin.ts`

Reply:
Nice catch! A common wrapper has been created which consists of legacy and new dependencies.